.github/actions/tests-linux-containerless/action.yaml

```diff
@@ -18,6 +23,8 @@ runs:
   using: "composite"
   steps:
   - uses: actions/checkout@v4
+    with:
+      ref: ${{ inputs.tests_ref }}
```

Comment:
Doing checkout with `_targeted` workflow (write GH token permissions) is too crazy, much much more risky than exposing some token with r/o public packages personal token.

Likely going to close this PR.